src/Sentry.Hangfire/Sentry.Hangfire.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Official Hangfire integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
+    <PackageTags>$(PackageTags);OpenTelemetry</PackageTags>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(FrameworkSupportsAot)' == 'true'">
+    <IsAotCompatible>true</IsAotCompatible>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Sentry\Sentry.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Hangfire.Core" Version="1.8.7" />
+    <PackageReference Include="Hangfire.MemoryStorage" Version="1.8.0" />
+    <PackageReference Include="Hangfire.AspNetCore" Version="1.8.7" />
```

Comment:
If this is going to depend on their ASP.NET Core package, we can even remove net462 because it's not going to work on .NET Framework.

Sucks to add more packages but ideally we'd have `Sentry.Hangfire` and `Sentry.Hangfire.AspNetCore` the latter depending on Sentry.AspNetCore if that helps with anything (like extension method `AddHangfire` on `SentryAspNetCoreOptions` or something to make it easier to add